PUBLISHING.md

```diff
@@ -5,71 +5,69 @@ the crates in this repository to [crates.io](https://crates.io/).
 
 **It is mostly intended for maintainers of the uefi-rs project.**
 
-## Bumping the crate versions
+## TL;DR
+- Each release should match the [`CHANGELOG.md`].
```

Comment:
We should write [`CHANGELOG.md`](CHANGELOG.md) if we want the link to work, otherwise Markdown will try to create a link to a file literally named `` `CHANGELOG.md` `` and fail.